tools/build.gradle

```diff
@@ -0,0 +1,17 @@
+
+plugins {
+  id 'java-gradle-plugin'
+}
+
+dependencies {
+  implementation "commons-io:commons-io:2.4"
+}
+
+gradlePlugin {
```

Comment:
Is there anything else that we have to add in order to make this plugin available for the users? I haven't published any plugins yet and I'm wondering if I have to add any new Maven repos in order to make it work.

Reply:
I'm not sure, as I don't have a lot of experience setting this up on the publishing side. I did test this by running the `install` task and then using it in one of my game projects via `mavenLocal()`. I don't know if there it anything else you need to check to make sure it is published automatically along with everything else because I haven't familiarized myself with the various scripts you have set up for publishing.

Reply:
Alright, I guess we'll figure it out once it's merged.